index.html

```diff
@@ -2309,6 +2309,112 @@ <h2 id="AudioDestinationNode">
+              member contains estimation of the audio stream position which is
+              currently being rendered by the
+              <a><code>AudioDestinationNode</code></a> (i.e. the current output
+              device audio stream position), in the same units and origin as
+              context's <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>;
+              the <a href=
+              "#widl-AudioTimestamp-performanceTime"><code>performanceTime</code></a>
+              member contains performance time estimation of the moment when
+              the stored audio stream position is rendered, in the same units
+              and origin as <code>performance.now()</code> (described in
+              [[!hr-time-2]]).
+            </p>
+            <p>
+              If the context's rendering graph has not yet processed a block of
+              audio, then <a><code>devicePosition</code></a> holds an
+              <code>AudioTimestamp</code> instance with both members containing
+              zero.
+            </p>
+            <p>
+              The value of <a><code>devicePosition</code></a> attribute can be
+              used to get performance time estimation for the slightly later
+              context's time value:
+            </p>
+            <pre class="highlight example">
+            function outputPerformanceTime(contextTime) {
+                var timestamp = destNode.devicePosition;
+                var elapsedTime = contextTime - timestamp.contextTime;
+                return timestamp.performanceTime + elapsedTime * 1000;
+            }
+            </pre>
+            <p>
+              In the above example the preciseness of the estimation depends on
+              how close the argument value is to the current output audio
+              stream position: the closer given <code>contextTime</code> to
+              <code>timestamp.contextTime</code>, the better preciseness of the
+              obtained estimation.
+            </p>
+            <p class="note">
+              The difference between the values of context's <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
+              and the <a href=
+              "#widl-AudioTimestamp-contextTime"><code>contextTime</code></a>
+              obtained from <a href=
+              "#widl-AudioDestinationNode-devicePosition"><code>devicePosition</code></a>
+              attribute value cannot be considered as a reliable output latency
+              estimation because <a href=
+              "#widl-BaseAudioContext-currentTime"><code>currentTime</code></a>
+              may be incremented at non-uniform time intervals, so <a href=
+              "#widl-AudioNode-latency"><code>latency</code></a> attribute
+              should to be used instead.
+            </p>
+          </dd>
+        </dl>
+      </section>
+      <section>
+        <h2>
+          AudioTimestamp
```

Comment:
Not sure what respec does, but should the definition of the AudioTimestamp dictionary come before the description of the attributes?